projects/rna/in-place-connection/package.json

```diff
@@ -0,0 +1,17 @@
+{
+	"name": "jetpack-in-place-connection",
+	"version": "1.0.0",
+	"description": "Jetpack In-Place Connection Component",
+	"main": "index.jsx",
```

Comment:
This should point to the built output of the package not the source code for the package.

For example, it should point to `build/index.js` if `build` is the output folder for the package's build process. If you choose not to do this, consumers of the package will have to transpile the package themselves, which is unusual. Typically when you install a package from `npm` you expect it to ship correct cjs and esm versions (depending on the intended usage of the package).

Reply:
Thanks for pointing this out, I removed the `main` entry for now.